Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_006.GitHubPat.sarif

```diff
@@ -74,6 +74,7 @@
             }
           ],
           "fingerprints": {
+            "AssetFingerprint/v1": "",
```

Comment:
>AssetFingerprint [](start = 13, length = 16)

Open an issue 'Add GitHub and AzureDevOps entries to AssetPlatform and emit platform data from personal access token rules."